css-brochure-form/styles.css

```diff
@@ -0,0 +1,74 @@
+#formHeading {
+  font-family: 'Brandon Grotesque';
+  font-style: normal;
+  font-weight: 500;
+  src: url('BrandonGrotesque-Medium.woff2') format('woff2');
+  text-transform: uppercase;
+  font-size: 20px;
+  color: #f64958;
+  margin-top: 15px;
+  text-align: center;
+  letter-spacing: 1px;
+}
+
+.formLabel {
+  font-family: 'Brandon Grotesque';
+  font-style: normal;
+  font-weight: 500;
+  src: url('BrandonGrotesque-Medium.woff2') format('woff2');
+  text-transform: uppercase;
+  font-size: 12px;
+  margin-left: 10px;
+}
+
+.inputBox {
+  font-family: 'AtlasGrotesk';
+  font-style: normal;
+  font-weight: 300;
+  src: url('AtlasGrotesk-Light-Web.woff2') format('woff2');
+  padding: 5px;
+  color: #b8b9ba;
+  border-radius: 4px;
+  border: 0.5px solid #b8b9ba;
+  margin: 1px;
+  width: 450px;
+  height: 35px;
+  display: block;
+}
+
+select {
+  font-family: 'Brandon Grotesque';
+  font-style: normal;
+  font-weight: 500;
+  src: url('BrandonGrotesque-Regular.woff2') format('woff2');
+  padding: 2px;
+  color: #2b2926;
+  border-radius: 4px;
+  border: 0.5px solid #b8b9ba;
+  margin: 1px;
+  background-color: #f2efe9;
+  width: 450px;
+  height: 35px;
+}
+
+button {
+  font-family: 'Brandon Grotesque';
+  font-style: normal;
+  font-weight: 450;
+  src: url('BrandonGrotesque-Regular.woff2') format('woff2');
+  background-color: #f64958;
+  color: white;
+  border-radius: 15px;
+  border: none;
+  text-align: center;
+  text-transform: uppercase;
+  padding: 5px 20px;
+  margin-left: 360px;
```

Comment:
Woah, that's a lot of margin! If you are using this to place the button on the right side of the form, you should instead apply `text-align: right` to the `div` element that the `button` is a child of.

Reply:
This still needs to be addressed.